testing/kuttl/e2e/delete/13-delete-cluster-and-check.yaml

```diff
@@ -25,16 +25,20 @@ commands:
       if [ -z "$REPLICA" ]; then exit 1; fi
 
       kubectl delete postgrescluster -n "${NAMESPACE}" delete-switchover
+      
+      kubectl wait pod/"${REPLICA}" --namespace "${NAMESPACE}" --for=delete --timeout=-1s
 
-      KILLING_PRIMARY_TIMESTAMP=$(
+      KILLING_REPLICA_TIMESTAMP=$(
         kubectl get events --namespace="${NAMESPACE}" \
-          --field-selector reason="Killing",involvedObject.fieldPath="spec.containers{database}",involvedObject.name="${PRIMARY}" \
+          --field-selector reason="Killing",involvedObject.fieldPath="spec.containers{database}",involvedObject.name="${REPLICA}" \
           --output=jsonpath={.items..firstTimestamp}
       )
-      
-      KILLING_REPLICA_TIMESTAMP=$(
+
+      kubectl wait pod/"${PRIMARY}" --namespace "${NAMESPACE}" --for=delete --timeout=-1s
```

Comment:
🤔 What does this command do if the pod is already gone?

Reply:
`wait --for=delete` seems to have three outcomes: waiting (until timeout), observing the delete (returns "condition met"), failing to find the object (returns immediately, no error):
```➜  ~ k wait pod/task-pv-pod --for=delete
error: timed out waiting for the condition on pods/task-pv-pod
➜  ~ k wait pod/task-pv-pod --for=delete
pod/task-pv-pod condition met
➜  ~ k wait pod/task-pv-pod --for=delete
➜  ~ ```

So seems safe to wait for a non-existent object to be deleted, but open to discussing and maybe going over the code.